src/traces/heatmap/hover.js

```diff
@@ -46,8 +48,8 @@ module.exports = function hoverPoints(pointData, xval, yval, hovermode, contour)
             return;
         }
     }
-    else if(Fx.inbox(xval - x[0], xval - x[x.length - 1]) > Fx.MAXDIST ||
-            Fx.inbox(yval - y[0], yval - y[y.length - 1]) > Fx.MAXDIST) {
+    else if(Fx.inbox(xval - x[0], xval - x[x.length - 1]) > MAXDIST ||
+            Fx.inbox(yval - y[0], yval - y[y.length - 1]) > MAXDIST) {
```

Comment:
sorry about that @etpinard - as a bit of karmic penance, I fixed this myself in #673 then had to undo my fix when I merged master and got your fix too!